telemetry/admin-metrics-export/src/main/scala/io/buoyant/telemetry/admin/AdminMetricsExportTelemeter.scala

```diff
@@ -51,16 +51,14 @@ class AdminMetricsExportTelemeter(
 
   private[this] def run0() = {
     val task = timer.schedule(snapshotInterval) {
-      summarySnapshots = snapshotHistograms(metrics)
+      snapshotHistograms(metrics)
     }
```

Comment:
i think ideally it may be preferable to move this timer out into wherever the MetricsTree is configured, but definitely not a blocker